tests/DataBuilderTest.php

```diff
@@ -815,6 +815,11 @@ public function testFramesOrder()
         ));
         $frames = $dataBuilder->makeFrames(new \Exception(), false);
         $this->assertEquals('<main>', $frames[count($frames)-1]->getMethod());
+        $this->assertEquals(
+            "/home/ubuntu/workspace/tests/DataBuilderTest.php",
```

Comment:
What is this? This doesn't pass locally now

Reply:
Fixed this with PR https://github.com/rollbar/rollbar-php/pull/260 and merged with master. Travis CI is passing now.